static/js/stream_data.js

```diff
@@ -872,7 +871,7 @@ exports.initialize = function (params) {
         We only use it in this function to populate other
         data structures.
     */
-
+    // Aqui que eu encontro as web_public e todas as outras streams
```

Comment:
Ingles, por favor. :)  Which bucket do the web-public subscriptions go into?

Reply:
I'm sorry, @showell ! I pushed this yesterday just because it was easier to compare my changes after I rebased with master. I was trying to find out why I was getting that error I mentioned about the spinner on subscriptions, that turned out to be introduced in another commit.

I should have left a comment saying that and forgot, I will clean it up :)
Thanks for the review!